build-info-extractor/src/test/java/org/jfrog/build/extractor/executor/CommandExecutorTest.java

```diff
@@ -61,4 +65,20 @@ public void testExeCommand() {
             fail(ExceptionUtils.getRootCauseMessage(e));
         }
     }
+
+    @Test
+    public void testExeCommandWithSpaces() throws IOException, InterruptedException {
+        Path tmpDir = Files.createTempDirectory("testEscapeSpacesInPath");
+        try {
+            Path testDir = Files.createDirectories(tmpDir.resolve("Kuiil space"));
+            File executableFile = Files.createFile(testDir.resolve("ship.bat")).toFile();
+            assertTrue(executableFile.setExecutable(true));
+
+            CommandExecutor commandExecutor = new CommandExecutor(executableFile.getAbsolutePath(), null);
+            CommandResults results = commandExecutor.exeCommand(null, new ArrayList<>(), new ArrayList<>(), null);
```

Comment:
It appears that we're running here an empty command. I suggest we actually try run something here, like a shell sceipt that echo's "hello" and then check the result.